pkgs/top-level/python-packages.nix

```diff
@@ -10582,7 +10582,7 @@ self: super: with self; {
 
   s3-credentials = callPackage ../development/python-modules/s3-credentials { };
 
-  sabyenc3 = callPackage ../development/python-modules/sabyenc3 { };
+  sabctools = callPackage ../development/python-modules/sabctools { };
```

Comment:
Also add an alias for sabyenc3 in `python-aliases.nix`.

Reply:
As I mentioned above, I'd argue this would be better off as a throw, simply because it's a breaking change because of the rename etc.

Unless you think different?

Reply:
Making it a throw is probably ok.